gf-guide/tools.md

```diff
@@ -141,7 +141,7 @@ Fontbakery is a quality assurance (QA) testing tool. It runs a series of checks
 To install Fontbakery run the command:
 
 ``` code
-pip install fontbakery
+pip3 install -U fontbakery'[freetype]'
```

Comment:
`pip3 install -U fontbakery'[googlefonts]'` will install all dependencies needed to run checks from the Google Fonts profiles, which also includes freetype. Doing it this way will guarantee that, in the future, any additional dependencies will also be installed, as the Google Fonts profile continues evolving.

The same is valid for any other vendor-specific profile. There's one extra for each of them, so that installing is straightforward.

Reply:
oh! But what I just said is only valid for FontBakery version 0.9.0a1 or later (as of today it has not yet landed on a stable release, but only the most recent pre-releases of v0.9.0).

Reply:
So, do we wait for the stable version to land before updating the installation instructions? Or should we merge this here and then update it again when the new release happens? I would opt for the latter.

Reply:
yeah. Merge it now and update later.

Reply:
Okay, please approve it to unblock the merge.